### ISSUE TITLE
[BUG] The web server seems to be incorrectly configured for cookies required for PHP sessions!

### ISSUE THREAD

**Issue by hervehobbes** [FreshRSS/FreshRSS#5973]
Hello,

**Describe the bug**
I have the error message "The web server seems to be incorrectly configured for cookies required for PHP sessions!"
Just after finishing installation.

**To Reproduce**
1. Install FreshRss from latest.zip
2. Configure it
3. Try to access it

**Expected behavior**
I should access FreshRss

**Environment information (please complete the following information):**
- Ubuntu 22.04.3
- PHP 8.1.2-1ubuntu2.14 (cli)
- mysql  Ver 8
- 
**Additional context**
I'm not using Docker

Content of /freshrss/data/users/_/
[warning] --- Invalid session during login for user=XXXX, nonce=

Thank you for any help.

**Comment by Alkarex** [comment]
Hello,
What Web server, and do you use any proxy?

As the message tells, there seems to be a problem with your PHP sessions. Try to create the following `./FreshRSS/p/i/session.php` and try to access `https://freshrss.example.net/i/session.php`:

```php
<pre>
<?php
session_start();
$_SESSION['count'] = ($_SESSION['count'] ?? 0) + 1;
var_dump($_SESSION);
```

It should increment by one each time you refresh the page. If that does not work, there is a wrong session configuration in your PHP setup, independent of FreshRSS.

A wrong session can also be due to wrong cookie path. Have you maybe installed more than once at different paths? You might have a wrong `base_url` in `./FreshRSS/data/config.php`. Back-up and remove that file before you try to install again.

**Comment by hervehobbes** [comment]
Hello,

I'm using Apache 2.4 and don't have any proxy.

Here is the result of the Php code:
```
array(1) {
  ["count"]=>
  int(1)
}
```

After 5 refresh here is the result:

```
array(1) {
  ["count"]=>
  int(5)
}
```

I renamed /data/config.php into config.php.old and then relaunched the site but I have:

> Error during context user init!

And nothing new in log.txt.

The base_url parameter in config.php seems correct.

I have activated error_log in php.ini but can't see anything in the error log.
